adops/setting-up-prebid-for-amp-in-dfp.md

```diff
@@ -11,17 +11,20 @@ nav_section: tutorials
 
 <div class="bs-docs-section" markdown="1">
 
-# Setting up Prebid for AMP in DFP
+# Setting up Prebid for AMP in DFP (Pre-release)
 {: .no_toc}
 
-{% include amp-deprecation-warning.md %}
+{: .alert.alert-warning :}
+The instructions on this page refer to the new "[AMP RTC][RTC-Overview] over [Prebid Server][PBS]" method which is planned for the [1.4 release](https://github.com/prebid/Prebid.js/releases/tag/1.4.0).  It doesn't actually work yet.  This page will be updated when the release goes out.
```

Comment:
So nothing here actually depends on the prebid.js release anymore. So having this warning doesn't make much sense - you could warn about general availability of the endpoint on prebid server

Reply:
Thanks, removed in ccd88b5

Removed the warning whole hog for now for expediency, can add another warning back in as needed.

Also removed the "Pre-release" text, since as you're saying that is not an issue.